Packs/Cisco-umbrella-enforcement/TestPlaybooks/CiscoUmbrellaEnforcement-Test.yml

```diff
@@ -0,0 +1,280 @@
+id: Cisco Umbrella Enforcement-Test
+version: -1
+name: Cisco Umbrella Enforcement-Test
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: d3fa006e-3d3c-4dfd-89e4-58681de95c5d
+    type: start
+    task:
+      id: d3fa006e-3d3c-4dfd-89e4-58681de95c5d
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "1":
+    id: "1"
+    taskid: 2b57d819-ee2d-4daa-8fa1-c980f2ef8f3d
+    type: regular
+    task:
+      id: 2b57d819-ee2d-4daa-8fa1-c980f2ef8f3d
+      version: -1
+      name: DeleteContext
+      script: DeleteContext
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "2"
+    scriptarguments:
+      all:
+        simple: "yes"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 195
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "2":
+    id: "2"
+    taskid: 63a0c12d-50f5-4098-8171-afca23bb31c6
+    type: regular
+    task:
+      id: 63a0c12d-50f5-4098-8171-afca23bb31c6
+      version: -1
+      name: umbrella-domain-event-add
+      script: '|||umbrella-domain-event-add'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      alert_time:
+        simple: "2013-02-08T09:30:26.0Z"
+      destination_domain:
+        simple: test1.com
+      destination_url:
+        simple: test1.com
+      device_id:
+        simple: ba6a59f4-e692-4724-ba36-c28132c761de
+      device_version:
+        simple: 13.7a
```

Comment:
please consider testing more of the args that you have recently added, or create 1 more `umbrella-domain-event-add` task

Reply:
I dont see the value in that, as this command only adds the new domain to the domains list, all the other arguments of the command are not saved within any api call so cant be checked.
The only reason that the args are inserted is because they are mandatory for this call.